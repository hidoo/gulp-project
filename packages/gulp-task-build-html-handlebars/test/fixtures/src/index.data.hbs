---
seo:
  title: "page title{{#if data.site.title}} | {{ data.site.title }}{{/if}}"
  description: "page description.{{#if data.site.description}}{{ data.site.description }}{{/if}}"
---
this is contents.
{{this.seo.title}}
{{this.seo.description}}
