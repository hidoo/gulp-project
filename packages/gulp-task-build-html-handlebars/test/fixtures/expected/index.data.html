this is contents.
page title | site title
page description.site description.
